druid/src/window.rs

```diff
@@ -527,7 +524,7 @@ impl<T: Data> Window<T> {
         let mut ctx = PaintCtx {
             render_ctx: piet,
             state: &mut state,
-            widget_state: &widget_state,
+            widget_state: &mut widget_state,
```

Comment:
What's the reason for this `mut` here? Seems not needed, the code certainly compiles without it.

Reply:
Jup. I forgot to change that back.